packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.scss

```diff
@@ -151,10 +151,23 @@ $DropDown-item-height: 36px;
   text-overflow: ellipsis;
 }
 
+.ms-Dropdown-callout {
+  box-shadow: 0 0px 15px -5px rgba(0, 0, 0, 0.4);
+  border: 1px solid $ms-color-neutralLight;
+}
+
+.ms-Dropdown-panel {
+  .ms-Panel-main {
+    box-shadow: 0 0px 15px -5px rgba(0, 0, 0, 0.4);
```

Comment:
ah, I'm using the same shadow that's used in the dropdown callout. I'm Ok using either.